lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -1809,17 +1809,122 @@ bool SwiftLanguageRuntimeImpl::IsValidErrorValue(ValueObject &in_value) {
   return true;
 }
 
+static bool
+CheckIfAnyTypesDeclaredInLLDB(const swift::reflection::TypeRef *typeref) {
```

Comment:
Why "any" types? Aren't we only interested in the outermost protocol type? Or does "any" mean any of the types inside a protocol composition?
It might help to explain what exactly this function does in a doxygen comment.

Reply:
It means any of the types inside the composition. So far, I've only seen single protocols being passing in to the `GetDynamicTypeAndAddress_Protocol` function, but I'm being very liberal here just in case.